pages/tutorials/[slug].js

```diff
@@ -5,33 +5,38 @@ import { marked } from 'marked'
 import styled from 'styled-components'
 import DynNavbar from '../../components/DynNavbar'
 import Footer from '../../components/Footer'
+import { serialize } from 'next-mdx-remote/serialize'
+import { MDXRemote } from 'next-mdx-remote'
+import SyntaxHighlighter from 'react-syntax-highlighter'
```

Comment:
I need to have a look at these new libraries.
In general, the less external dependencies the better, but if they are reliable and making a good job it is a nice idea :)